src/cargo/ops/cargo_package.rs

```diff
@@ -405,6 +429,23 @@ fn warn_on_nonexistent_file(
     ))
 }
 
+fn warn_custom_build_file_not_in_package(
+    pkg: &Package,
+    path: &Path,
+    target: &Target,
+    ws: &Workspace<'_>,
+) -> CargoResult<()> {
+    let msg = format!(
+        "the source file of {:?} target `{}` doesn't appear to be a path inside of the package.\n\
+        It is at {}, whereas the root the package is {}.\n\
```

Comment:
for cosistency, could we add backticks here
```suggestion
        It is at `{}`, whereas the root the package is `{}`.\n\
```